docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -304,6 +322,101 @@ NOTE: You must use your cluster host domain name when creating your URL.
     ✓  Successfully deleted component
 ----
 
+== Deploying a Quarkus Application on Kubernetes/Openshift cluster
+
+In this example we will be deploying a https://github.com/odo-devfiles/quarkus-ex[quarkus component] that uses GraalVM and JDK1.8+.
+
+. Download the example quarkus component. 
++
+[source,sh]
+----
+ $ git clone https://github.com/odo-devfiles/quarkus-ex && cd quarkus-ex
+----
+
+. Create an quarkus odo component 
++
+[source,sh]
+----
+   $ odo create java-quarkus myquarkus
+   Experimental mode is enabled, use at your own risk
+
+   Validation
+    ✓  Checking devfile compatibility [195728ns]
+    ✓  Creating a devfile component from registry: DefaultDevfileRegistry [170275ns]
+    ✓  Validating devfile component [281940ns]
+
+    Please use odo push command to create the component with source deployed
+----
+
+. Create a URL in order to access the deployed component:
++
+* Run below for Kubernetes cluster
++
+[source,sh]
+----
+ $ odo url create --host my-cluster.info
+  ✓  URL myquarkus-8080.my-cluster.info created for component: myquarkus
+
+ To apply the URL configuration changes, please use odo push
+----
++
+
+NOTE: You must use your cluster host domain name when creating your URL.
```

Comment:
See other commments above, perhaps we can change these sections to say: "If deploying to Kubernetes, you must provide a cluster host name with --host" or something like that.